src/start.js

```diff
@@ -5,6 +5,6 @@ setupProxy(__dirname)(app);
 app.listen(3000);
 
 /**
- * YOU CODE SHOULD NOT GO HERE! please edit ./backend/_handler.js instead.
+ * YOUR CODE SHOULD NOT GO HERE! please edit ./backend/_handler.js instead.
```

Comment:
Opened https://github.com/reshufflehq/reshuffle/pull/317 to get rid of this file.